lib/helpers.js

```diff
@@ -5,18 +5,15 @@
 
 const base58 = require('base58-universal');
 const bedrock = require('bedrock');
-const brZCapStorage = require('bedrock-zcap-storage');
 const {config, util: {BedrockError}} = bedrock;
-const {promisify} = require('util');
 const crypto = require('crypto');
+const edvs = require('./storage/edvs.js');
+const logger = require('./logger');
+const {meters} = require('bedrock-meter-usage-reporter');
+const {promisify} = require('util');
 const getRandomBytes = promisify(crypto.randomBytes);
 
-exports.getEdvId = ({localId} = {}) => {
-  exports.assert128BitId(localId);
-  const {baseUri} = config.server;
-  const baseStorageUrl = `${baseUri}${config['edv-storage'].routes.basePath}`;
-  return `${baseStorageUrl}/${localId}`;
-};
+exports.SERVICE_TYPE = 'edv';
```

Comment:
This seems to be a better fit for a `constants` file.

Reply:
Yeah, for another refactoring time. Out of time here now.